lnrpc/routerrpc/routing_config.go

```diff
@@ -7,28 +7,16 @@ import (
 )
 
 // RoutingConfig contains the configurable parameters that control routing.
+//
+//nolint:lll
 type RoutingConfig struct {
+	// ProbabilityEstimatorType sets the estimator to use.
+	ProbabilityEstimatorType string `long:"estimator" choice:"apriori" choice:"bimodal" description:"Probability estimator used for pathfinding." `
+
 	// MinRouteProbability is the minimum required route success probability
 	// to attempt the payment.
 	MinRouteProbability float64 `long:"minrtprob" description:"Minimum required route success probability to attempt the payment"`
 
-	// AprioriHopProbability is the assumed success probability of a hop in
-	// a route when no other information is available.
-	AprioriHopProbability float64 `long:"apriorihopprob" description:"Assumed success probability of a hop in a route when no other information is available."`
-
-	// AprioriWeight is a value in the range [0, 1] that defines to what
-	// extent historical results should be extrapolated to untried
-	// connections. Setting it to one will completely ignore historical
-	// results and always assume the configured a priori probability for
-	// untried connections. A value of zero will ignore the a priori
-	// probability completely and only base the probability on historical
-	// results, unless there are none available.
```

Comment:
The breaking change here is worth pointing out in isolation in the release notes.